commerce-add-ons/payment-providers/stripe/release-notes.md

```diff
@@ -16,6 +16,9 @@ In this section, you can find the release notes for each version of Stripe Payme
 
 <summary>Version 13</summary>
 
+#### 13.1.0 (December 28th 2023)
```

Comment:
can you please add a link to this as well, similar to https://github.com/umbraco/UmbracoDocs/pull/5737/files? 😊 
![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/f018eafb-92ed-4a8a-bc51-3423ac6494b0)

Reply:
Hi @umbracotrd I was wondering if you have seen my latest message and if you are able to make add the label for stripe and add a link to it here?  😊